Java/Java.sublime-syntax

```diff
@@ -598,20 +573,56 @@ contexts:
 
   parens:
     - match: \(
+      scope: punctuation.section.parens.begin.java
       push:
+        - meta_scope: meta.parens.java
```

Comment:
It says `based on appropriate semantics`. What's the difference between group and parens? Java doesn't have a concept of a `group` so parens seemed appropriate.